Examples/Run/ReadCsv/GenericReadCsvExample.cpp

```diff
@@ -16,72 +16,75 @@
 #include "ActsExamples/Io/Csv/CsvPlanarClusterWriter.hpp"
 #include "ActsExamples/Options/CommonOptions.hpp"
 #include "ActsExamples/Printers/PrintHits.hpp"
+#include "ActsExamples/Printers/PrintParticles.hpp"
 #include "ActsExamples/Utilities/Options.hpp"
 
 #include <memory>
 
+using namespace ActsExamples;
+
 int main(int argc, char* argv[]) {
   GenericDetector detector;
 
   // setup and parse options
-  auto desc = ActsExamples::Options::makeDefaultOptions();
-  ActsExamples::Options::addSequencerOptions(desc);
-  ActsExamples::Options::addGeometryOptions(desc);
-  ActsExamples::Options::addMaterialOptions(desc);
-  ActsExamples::Options::addInputOptions(desc);
+  auto desc = Options::makeDefaultOptions();
+  Options::addSequencerOptions(desc);
+  Options::addGeometryOptions(desc);
+  Options::addMaterialOptions(desc);
+  Options::addInputOptions(desc);
   detector.addOptions(desc);
-
-  auto vm = ActsExamples::Options::parse(desc, argc, argv);
-  if (vm.empty()) {
+  auto vars = Options::parse(desc, argc, argv);
+  if (vars.empty()) {
     return EXIT_FAILURE;
   }
 
-  ActsExamples::Sequencer sequencer(
-      ActsExamples::Options::readSequencerConfig(vm));
+  Sequencer sequencer(Options::readSequencerConfig(vars));
 
   // Read some standard options
-  auto logLevel = ActsExamples::Options::readLogLevel(vm);
-  auto inputDir = vm["input-dir"].as<std::string>();
+  auto logLevel = Options::readLogLevel(vars);
+  auto inputDir = vars["input-dir"].as<std::string>();
 
   // Setup detector geometry
-  auto geometry = ActsExamples::Geometry::build(vm, detector);
+  auto geometry = Geometry::build(vars, detector);
   auto trackingGeometry = geometry.first;
   // Add context decorators
   for (auto cdr : geometry.second) {
     sequencer.addContextDecorator(cdr);
   }
 
   // Read particles from CSV files
-  auto particleReaderCfg =
-      ActsExamples::Options::readCsvParticleReaderConfig(vm);
-  particleReaderCfg.outputParticles = "particles";
-  sequencer.addReader(std::make_shared<ActsExamples::CsvParticleReader>(
-      particleReaderCfg, logLevel));
+  auto particleReader = Options::readCsvParticleReaderConfig(vars);
```

Comment:
But this is actually the config, not the reader. So doesn't the old name make more sense?

Reply:
I think this way is easier. It makes sense to name this `AlgorithmCfg` if there is also an `Algorithm` variable. Here, the algorithms are never visible since they are only created as part of the `.addAlgorithm(...)` call. So we can safely leave out the `...Cfg` and make everything a bit shorter.

Reply:
Huh. I think this kind of obfuscates what's happening here. These three characters **really** do not matter in code like this in my opinion.

Reply:
Anyway, it's gone from the diff now.